### PR TITLE
Working branch

### PR DESCRIPTION
**Pull request by mi4l**

### CODE REVIEW

File: mike_skalandunas/app/css/skeleton.css [hunk 1]
```diff
@@ -0,0 +1,418 @@
+/*
```
Comment (warmbowski) [inline mike_skalandunas/app/css/skeleton.css]: just curious if you looked for an npm module for this.

File: mike_skalandunas/package.json [hunk 2]
```diff
@@ -7,13 +7,28 @@
...
   },
   "dependencies": {
+    "bcrypt": "latest",
     "body-parser": "^1.14.1",
+    "eat": "^0.1.1",
     "express": "^4.13.3",
     "mongoose": "^4.2.5"
   },
   "devDependencies": {
+    "angular": "^1.4.8",
```
Comment (warmbowski) [inline mike_skalandunas/package.json]: dev? I think you'll be really sad if you pushed this to heroku.

File: mike_skalandunas/app/js/tapirs/controllers/tapirs_controller.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    };
+
+    $scope.reset = function(tapir) {
+      $scope.master = {};
+
+      $scope.reset = function() {
+        angular.copy($scope.master, $scope.user);
+      };
+
+      $scope.reset();
```
Comment (warmbowski) [inline mike_skalandunas/app/js/tapirs/controllers/tapirs_controller.js]: This doesn't look like it works right. I would assume, you would need to set $scope.master equal to a copy of the $scope.tapirs after the api call. Then a reset is just setting the edited array equal to the master.

**Comment by mskalandunas** [comment]
Yup!  Does not work.  I'm working on implementing that tonight.

**Pull request closed without merging**